check_run/check_run/doctype/check_run/check_run.json

```diff
@@ -22,6 +22,7 @@
   "pay_to_account",
   "section_break_9",
   "check_run_table",
+  "html_15",
```

Comment:
Please remove this field in favor of "check_run_table"

Reply:
This was actually the fix to make the section not disappear on save (having an additional empty html field in the same section)